src/browser/api/application.js

```diff
@@ -837,18 +861,27 @@ Application.setTrayIcon = function(identity, iconUrl, callback, errorCallback) {
     });
 };
 
-Application.setZoomLevel = function(identity, level) {
-    const app = coreState.appByUuid(identity.uuid);
+Application.setZoomLevel = function(identity, level, callback, errorCallback) {
+    const uuid = identity.uuid;
+    const appError = checkApplicationAvailability(uuid);
 
-    // set zoom level for each child window
-    app.children.forEach(function(childWindow) {
-        const childWindowIdentity = {
-            name: childWindow.openfinWindow.name,
-            uuid: childWindow.openfinWindow.uuid
-        };
-        Window.setZoomLevel(childWindowIdentity, level);
-    });
+    if (appError) {
+        errorCallback(new Error(appError));
```

Comment:
Need to check `typeof errorCallback === 'function'` just like `callback`

Reply:
Fixed it. Actually we don't need callback and errorcallback in this function.